<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleRunStatePage</class>
 <widget class="QWidget" name="VehicleRunStatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QLabel
{
	border:1px solid lightgray;
    color:white;
	font:16px
}</string>
  </property>
  <widget class="QLabel" name="LBLC11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/images/panto1up2down.png);	</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC13">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC14">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC15">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC16">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC17">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC18">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC19">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>395</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC23">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC29">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC27">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC28">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC24">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC22">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC25">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLC26">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>445</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLD41">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>335</y>
     <width>330</width>
     <height>165</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLD31">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>125</y>
     <width>330</width>
     <height>210</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBLD11">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>0</y>
     <width>330</width>
     <height>125</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>15</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
</string>
   </property>
   <property name="text">
    <string>司控器级位</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
</string>
   </property>
   <property name="text">
    <string>设定速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>85</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
</string>
   </property>
   <property name="text">
    <string>实际速度</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBLGrade">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLSetSpeed">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>45</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLSpeed">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_7">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>15</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
color:yellow;
font:14px;</string>
   </property>
   <property name="text">
    <string>级</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>85</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
color:yellow;
font:14px;</string>
   </property>
   <property name="text">
    <string>km/h</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>50</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
color:yellow;
font:14px;</string>
   </property>
   <property name="text">
    <string>km/h</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_10">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
color:yellow;
font:14px;</string>
   </property>
   <property name="text">
    <string>-1km/h</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_11">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>50</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
color:yellow;
font:14px;
</string>
   </property>
   <property name="text">
    <string>+1km/h</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
</string>
   </property>
   <property name="text">
    <string>速度设定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNPlus1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/down.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BTNMinus1">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/images/up.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange;
border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string>故障信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLFault1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLFault2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>185</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLFault3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLFault4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>235</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLFault5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLFault6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>285</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLFault7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL1_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:orange;
border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string>提示信息</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTips2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>395</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTips1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTips4">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>445</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTips3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>420</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBLTips5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>470</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid transparent;
font:14px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
